starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,58 @@
-<!DOCTYPE html>
+<!DOCTYPE HTML5>
 <html lang="en">
 <head>
   <title>Apple Pie Recipe</title>
+  <meta charset="UTF-8">
+  <link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
 <body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
 
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
+<div id="apple_pie"><p>Apple Pies</p></div>
+
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
+<p id="description">This was my grandmother's apple pie recipe. I have never seen another one
+quite like it. It will always be my favorite and has won me several first place
+prizes in local competitions. I hope it becomes one of your favorites as well!
+</p><br><br><br>
```

Comment:
this is not a clean solution to go onto the next line, try to come up with something else bro